### PR TITLE
Create first version of a CustomUser model

### PR DESCRIPTION
**Pull request by JakubWronka**
Add CustomUser model to the blog application

### CODE REVIEW

File: bestprojectever/blog/admin.py [hunk 1]
```diff
@@ -1,3 +1,5 @@
 from django.contrib import admin
+from django.contrib.auth.admin import UserAdmin
+from .models import CustomUser
 
-# Register your models here.
+admin.site.register(CustomUser, UserAdmin)
```
Comment (PythonMaciej) [inline bestprojectever/blog/admin.py:5]: Can I just ask you to register here Article model please (I have forgotten to do it on my PR)?

**Review by PythonMaciej — CHANGES_REQUESTED** [review]
Great job, best project ever!

File: bestprojectever/blog/admin.py (hunk 1 above)
Comment (JakubWronka) [reply bestprojectever/blog/admin.py:5]: sure, will be done in the next commit

Comment (PythonMaciej) [reply bestprojectever/blog/admin.py:5]: Well done, thanks

**Review by PythonMaciej — APPROVED** [review]
Well done

**Pull request merged**